modular-docs-manual/content/topics/using_text_snippets_or_text_fragments.adoc

```diff
@@ -17,13 +17,16 @@
 [NOTE]
```

Comment:
Is this note relevant? Don't these guidelines apply in all cases?

Reply:
Hi @emmurphy1, the existing content comes from this discussion - https://github.com/redhat-documentation/modular-docs/issues/18

Do you think the note is not relevant anymore? When I started with mod-docs, I inferred from this note that text-snippets are discouraged because it is a challenge to track them manually. Having a CMS alleviates that issue.